draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -598,6 +598,11 @@ As it mentioned in {{ne-component}} that state data and performance data are out
 
 Mostly, our inventory data model can cover the attributes from OpenConfig.
 
+# Terminology of Container
+
+In {{!RFC8348}}, container is defined as an identity of component class . According to the description, it is capable to container one or more removeable physical entities, e.g. a slot in a chassis is containing a board. It can be mapped to the term of 'holder' in other models.
+Besides, The terms of container in this document can be 'container component' or 'YANG container'. The understanding of the meaning of 'container' should be combined with context.
```

Comment:
I think the definition proposed is not totally aligned with the proposed text https://github.com/ietf-ivy-wg/network-inventory-yang/issues/13#issuecomment-1904342163 and as suggested in the comment https://github.com/ietf-ivy-wg/network-inventory-yang/issues/13#issuecomment-1904342163

Reply:
I think that the proposed text already exists in the definition section:

> Container:
> A hardware component class that is capable of containing one or more removable physical entities (e.g., a slot in a chassis is containing a board).
> 

I am wondering whether this appendix cannot be rewritten with a table providing a mapping between the terminology in [TMF_SD2-20] and the definitions in our model. It might be worthwhile referencing this appendix in section 3.3.